src/bulk/common.ts

```diff
@@ -1133,6 +1133,9 @@ export abstract class BulkOperationBase {
 
   /** Specifies a raw operation to perform in the bulk write. */
   raw(op: AnyBulkWriteOperation): this {
+    if (!op) {
+      throw new MongoInvalidArgumentError('Cannot call .raw() with an undefined operation');
```

Comment:
```suggestion
    if (op != null || typeof op != 'object') {
      throw new MongoInvalidArgumentError('Operation must be an object with an operation key');
```
I think this error would catch a few more error cases where someone passes in something truthy but the incorrect type altogether. Also, we try to phrase error messages in terms of what must be done to correct the code.

Reply:
upon changing the conditional to what you suggested, started getting a bunch of errors. Ive attached a screen shot of the error im getting to this message.
<img width="1728" alt="Screen Shot 2022-05-02 at 4 02 36 PM" src="https://user-images.githubusercontent.com/46140162/166317402-dfdc84ec-33fd-42f2-b6e8-4ce21948ba74.png">

Reply:
My mistake! I meant `op == null || ...`

Reply:
oooooohhh!